src/bounds.jl

```diff
@@ -146,3 +146,51 @@ a definite sign.
 
 """
 bound_taylor1(fT::TaylorModel1, I=domain(fT)::Interval) = bound_taylor1(polynomial(fT), I)
+
+"""
+    linear_dominated_bounder(fT::TaylorModel1, ϵ=1e-3::Float, max_iter=5::Int)
+
+Compute a tighter polynomial bound for the Taylor model `fT` by the linear
+dominated bounder algorithm. The linear dominated algorithm is applied until
+the bound of `fT` gets tighter than `ϵ` or the number of steps reachs `max_iter`.
+"""
+function linear_dominated_bounder(fT::TaylorModel1; ϵ=1e-3, max_iter=5)
+    d = 1.
+    Dn = fT.dom
+    Dm = Dn
+    Pm = Taylor1(copy(fT.pol.coeffs))
+    bound = interval(0.)
+    n_iter = 0
+    while d > ϵ && n_iter < max_iter
+        if n_iter == 0
+            x0 = mid(Dn) - mid(fT.x0)
+        else
+            x0 = mid(Dn) - mid(Dm)
+        end
+        c = mid(Dn)
+        Pm = Taylor1(copy(Pm.coeffs))
+        update!(Pm, x0)
+        linear = Taylor1(copy(Pm.coeffs))
+        linear[2:end] = zero(linear[0])
+        non_linear = Taylor1(copy(Pm.coeffs))
+        non_linear[0:1] = zero(non_linear[0])
```

Comment:
An alternative for these lines is:
```julia
linear = Taylor1(Pm.coeffs[1:2], Pm.order)
Taylor1(Pm.coeffs[1:2], Pm.order)
```
This may even be slightly more performant...

Reply:
Thanks for the suggestions, is there an equivalent statement for the non-linear part?
I'm thinking in something like
```julia
non_linear = Taylor1(Pm.coeffs[3:end], Pm.order)
```
but this doesn't give what I want. Is there a way to specify an "offset" on the coefficients?

Reply:
There is no option to specify an offset. But, since `linear` is defined (and of the proper `order`), you could simply define:
```julia
non_linear = Pm - linear
```